norns/wifi-files.md

```diff
@@ -77,16 +77,16 @@ Here's the `dust` file tree:
 
 ```
 dust/
-  audio/          -- audio files
-    tape/             -- tape recordings
-    ...
-  code/           -- contains scripts and engines
-    awake/
-    mlr/
-    ...
-    we/
-  data/           -- contains user data created by scripts
-    awake/            -- for example, pset data
+audio/-- audio files
```

Comment:
this whitespace was actually for legibility, can you restore it?

Reply:
oh weird. must've been an IDE or copy paste issue. yes i'll fix.